modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -91,8 +91,12 @@ def claimant
       )
     end
 
+    def non_veteran_claimant?
+      claimant.signing_appellant?
+    end
```

Comment:
I added this new method because I was wishing for it in #15661 - if this looks like an OK addition, I can use it there too after merging this

Reply:
Nice addition!